### ISSUE TITLE
Just updated from 0.2 something to latest

### ISSUE THREAD

**Issue by simonrb2000** [nfarina/homebridge#590]
Hi, just updated and before I updated it was working fine. Now I get Error: Not a valid username: DD: 33: 4E: F4: DF: 31 . Must be 6 pairs of colon-separated hexadecimal chars (A-F 0-9), like a MAC address.

I have changed the 31 to a 32 to see if that made a difference and it doesn't. Below is my config file. Any ideas?? Cheers

{
	"bridge": {
		"name": "Homebridge",
		"username": "DD: 33: 4E: F4: DF: 31 ",
		"port": 51927,
		"pin": "031-45-154"
	},
	"description": "ConfigurationfileforDomoticzplatform.",
	"platforms": [{
		"platform": "Domoticz",
		"name": "Domoticz",
		"server": "127.0.0.1",
		"port": "8080",
		"roomid": "4",
		"loadscenes": "0"
	}]
}

**Comment by PeteLawrence** [comment]
Try removing the spaces from the username and you should be good to go. I'm not sure however what effect changing the username will have on your paired accessories though.

**Comment by simonrb2000** [comment]
Hi, I tried that and made no difference, When I use the JSON website to check it puts the spaces back in. I have even tried changing the numbers and letters and still get the same error. Whats strange it is the same config file I was using before I updated homebridge.

Any other ideas?

Cheers

**Comment by AppleTechy** [comment]
@simonrb2000 Your Mac address needs to have numerical value inside of an aplabetical character in certain places. Try something like `D3:33:4E:F4:DF:31`

**Comment by simonrb2000** [comment]
Hey, thanks for the reply. I got it working by removing all the spaces. JSON website was what threw me as that put spaces in. I removed the spaces and it still didn't work, didn't notice the space at the end DF:31 ", when I changed it to DF:31", it came back to life. I had tried that before but the text editor I use on Mac doesn't work with " so had to use JSON website but not check it as it would always add spaces. 

Any how's it's working great at the mo so thank you.
